modules/installation-registry-storage-non-production.adoc

```diff
@@ -36,3 +36,23 @@ Error from server (NotFound): configs.imageregistry.operator.openshift.io "clust
 ----
 +
 Wait a few minutes and run the command again.
+
+. Ensure that your registry is set to managed to enable building and pushing of images.
++
+* Run:
++
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+Then, change the line
```

Comment:
I would consider restructuring this step to be part of the numbered procedure.

    . Change the line `managementState: Removed` to `managementState: Managed`.

Reply:
Same thing here: I don't have a clean way to separate the "change the line" step while keeping it under the scope of "Ensure that your registry is set to managed...".